confmap/provider/httpprovider/provider.go

```diff
@@ -0,0 +1,76 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package httpprovider
+
+import (
+	"context"
+	"fmt"
+	"io/ioutil"
+	"strings"
+
+	"go.opentelemetry.io/collector/confmap"
+	"go.opentelemetry.io/collector/confmap/provider/internal"
+
+	"net/http"
+)
+
+const (
+	schemeName    = "http"
+	httpGetPrefix = "http://"
```

Comment:
Unused? and not very helpful of a const either

Reply:
Thanks Eric, fixed.